6_drivers_fetch.yml

```diff
@@ -16,19 +16,31 @@ targets:
     depends:
       - new_cubes
       # - append_cubes # how to make a task plan that can be empty?
-      - new_feathers
+      - "6_drivers/log/feather/6_fetch_new_feather_tasks.ind"
 
   replete_time_range:
-    command: c(t0=I(0), t1 = I(999))#346848))
+    command: c(t0=I(0), t1 = I(346848))
 
   append_time_range:
     command: nldas_times_diff(replete_time_range, '6_drivers/in/nldas_times.rds')
 
   nccopy_time_stride:
     command: c(I(25000))
 
+  cube_data_dir:
+    command: c(I("6_drivers/out/nc"))
+
+  cube_ind_dir:
+    command: c(I("6_drivers/log/nc"))
+
+  feather_data_dir:
```

Comment:
The other names - `cube`, `cell` - describe spatial units, but `feather` only gives the format. These `feather` files seem to be `cell` files...are there other `cell` files you're distinguishing from by using the name `feather` instead of `cell`?

Reply:
no, you're right. This is a spot where I wasn't consistent with the naming